specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-12-01-preview/examples/Migrations_Cancel.json

```diff
@@ -0,0 +1,58 @@
+{
+  "parameters": {
+    "api-version": "2023-12-01-preview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "testrg",
+    "targetDbServerName": "testtarget",
+    "migrationName": "testmigration",
+    "migrationMode": "Online",
+    "parameters": {
+      "properties": {
+        "cancel": "True"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "migrationId": "d3ceacbb-a5fd-43dc-a9db-6022b5154856",
+          "currentStatus": {
+            "state": "Canceled",
+            "currentSubStateDetails": {
+              "currentSubState": "Completed"
+            }
+          },
+          "migrationMode": "Online",
```

Comment:
Will Cancel response include migrateRoles

Reply:
Yes, it will. 
@nasc17 FYI.